src/commands/promote.js

```diff
@@ -1,24 +1,65 @@
 const colors = require('colors/safe');
 const utils = require('../utils');
 
+const hasCancelled = answer => (answer.toLowerCase() === 'n' || answer.toLowerCase() === 'no' || answer.toLowerCase() === 'cancel');
+const hasAccepted = answer => (answer.toLowerCase() === 'y' || answer.toLowerCase() === 'yes');
+
 const promote = (context, version) => {
   if (!version) {
     context.line('Error: No deploment/version selected...\n');
     return Promise.resolve();
   }
 
   let appId = 0;
+  let app, changelog;
 
   return utils.checkCredentials()
-    .then(() => utils.getLinkedApp())
-    .then((app) => {
+    .then(() => Promise.all([
+      utils.getLinkedApp(),
+      utils.getVersionChangelog(version),
+    ]))
+    .then(([foundApp, foundChangelog]) => {
+      app = foundApp;
+      changelog = foundChangelog;
+
       context.line(`Preparing to promote version ${version} of your app "${app.title}".\n`);
+
+      if (changelog) {
+        context.line(colors.green(`Changelog found for ${version}!`));
+        context.line(`\n---\n${changelog}\n---\n`);
+        return Promise.resolve('y');
```

Comment:
Actually, I was thinking even if we found a changelog, the user may want to preview the changelog before they submit it for promotion. For example, a user may forget to put a blank line between `## 1.0.0` and its first changelog item. They may want to go back and fix it. So I think it's nice to ask the user if they want to proceed to promote with the changelog we parse out from `CHANGELOG.md`. What do you think?

Reply:
Oh, that's a good idea. I'll add that tomorrow then!